### PR TITLE
Add mysql create.sql

### PR DESCRIPTION
**Pull request by vhpoet**

### CODE REVIEW

File: src/sql/mysql/create.sql [hunk 1]
```diff
@@ -0,0 +1,68 @@
+create table if not exists L_ACCOUNTS (
+ACCOUNT_ID int not null auto_increment primary key,
+NAME varchar(255),
+BALANCE numeric(32,16) check (BALANCE >= MINIMUM_ALLOWED_BALANCE),
```
Comment (clark800) [inline src/sql/mysql/create.sql]: MySQL does not enforce check constraints, which is why we removed MySQL support

**Comment by justmoon** [comment]
Would `UNSIGNED` work instead? http://dev.mysql.com/doc/refman/5.7/en/numeric-type-attributes.html

**Comment by clark800** [comment]
> Would UNSIGNED work instead?

Probably not, it looks like that just clamps the value to zero rather than generating an error: http://stackoverflow.com/a/8794634

Also, currently we are not shifting the balance so the balance value in the database can still go negative.

**Comment by vhpoet** [comment]
Closing for now, until we get a better idea on how to proceed with this.

**Pull request closed without merging**